Project.toml

```diff
@@ -9,13 +9,17 @@ ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 FastGaussQuadrature = "442a2c76-b920-505d-bb47-c5924d526838"
 FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
+ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 GPLikelihoods = "6031954c-0455-49d7-b3b9-3e1c99afaf40"
 KLDivergences = "3c9cd921-3d3f-41e2-830c-e020174918cc"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Optim = "429524aa-4258-5aef-a3af-852621145aeb"
```

Comment:
Used by `optimize_elbo`, which I modelled after https://github.com/willtebbutt/ConjugateComputationVI.jl/blob/master/src/approximate_inference.jl#L248 - though arguably that shouldn't be part of ApproximateGPs themselves. Would be nice to expose the warmstart option without people having to rewrite it in their downstream code. Maybe just a higher-order function from `build_latent_gp` to `objective_with_warmstart`:)

Reply:
did just that